internal/cloud/awscloud/awscloud.go

```diff
@@ -581,3 +581,18 @@ func (a *AWS) S3ObjectPresignedURL(bucket, objectKey string) (string, error) {
 	logrus.Info("[AWS] 🎉 S3 Presigned URL ready")
 	return url, nil
 }
+
+func (a *AWS) MarkS3ObjectAsPublic(bucket, objectKey string) error {
+	logrus.Infof("[AWS] 👐 Making S3 object public %s/%s", bucket, objectKey)
```

Comment:
I know that other methods in this file log messages as well, but I find it weird in a "library" implementation. The caller of this method could to it id needed (log before calling it and log message based on the fact if an `error` is returned. This is probably not something worth changing at this point, but I wanted to call this out to see if I'm the only one 😄

Reply:
Agreed, alternatively, we can pass logger inside the AWS object.